scripts/oqs-openssl-cmssign.sh

```diff
@@ -18,6 +18,13 @@ else
    exit -1
 fi
 
+openssl_version=$($OPENSSL_APP version)
+
+if [[ "$openssl_version" = "OpenSSL 3.0."* ]]; then
+        echo "Skipping CMS test for OpenSSL 3.0"
```

Comment:
I'm a bit rusty on bash but is this actually how you do string matching in bash? Won't the `*` try to do file name globbing?

Reply:
> Won't the * try to do file name globbing?

AFAIK not when using double brackets. Besides, it worked :-) Here's an independent opinion from StackOverflow: https://stackoverflow.com/questions/229551/how-to-check-if-a-string-contains-a-substring-in-bash . After having read that, though, I noticed the single "=" -- now fixed.